### PR TITLE
KAFKA-17013 RequestManager#ConnectionState#toString() should use %s

### PR DESCRIPTION
**Pull request by dujian0068**
[KAFKA-17013 RequestManager#ConnectionState#toString() should use %s](https://issues.apache.org/jira/browse/KAFKA-17013)

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by chia7712 — APPROVED** [review]
LGTM

File: raft/src/main/java/org/apache/kafka/raft/RequestManager.java [hunk 1]
```diff
@@ -372,7 +372,7 @@ void onRequestSent(long correlationId, long timeMs) {
         @Override
         public String toString() {
             return String.format(
-                "ConnectionState(node=%s, state=%s, lastSendTimeMs=%d, lastFailTimeMs=%d, inFlightCorrelationId=%d)",
+                "ConnectionState(node=%s, state=%s, lastSendTimeMs=%d, lastFailTimeMs=%d, inFlightCorrelationId=%s)",
```
Comment (chia7712) [inline raft/src/main/java/org/apache/kafka/raft/RequestManager.java:375]: `inFlightCorrelationId` is `OptionalLong`, so its `toString` includes the prefix "OptionalLong". Could you please do a bit refactor to remove that?

Comment (dujian0068) [reply raft/src/main/java/org/apache/kafka/raft/RequestManager.java:375]: Of course, it has been modified.

File: raft/src/main/java/org/apache/kafka/raft/RequestManager.java [hunk 2]
```diff
@@ -372,12 +372,12 @@ void onRequestSent(long correlationId, long timeMs) {
...
                 node,
                 state,
                 lastSendTimeMs,
                 lastFailTimeMs,
-                inFlightCorrelationId
+                inFlightCorrelationId.isPresent() ? inFlightCorrelationId.getAsLong() : null
```
Comment (chia7712) [inline raft/src/main/java/org/apache/kafka/raft/RequestManager.java]: how about using `undefined` to replace "null"?

Comment (dujian0068) [reply raft/src/main/java/org/apache/kafka/raft/RequestManager.java]: You are right, this makes it easier to understand
it has been modified.

**Comment by chia7712** [comment]
@dujian0068 thanks for contribution

**Pull request merged**